<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardSeeker</class>
 <widget class="QDialog" name="DashboardSeeker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Job Seeker Dashboard - Job Matching System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog { background-color: #f0f8ff; border: 1px solid #A3BFFA; font-family: Arial, sans-serif; }
    QLabel { font: 14px "Arial"; color: #2e2e2e; font-weight: bold; padding: 5px; }
    QTableWidget { background-color: #FFFFFF; border: 1px solid #A3BFFA; border-radius: 5px; font: 14px "Arial"; alternate-background-color: #f5f5f5; gridline-color: #ddd; }
    QTableWidget::item { padding: 5px; }
    QHeaderView::section { background-color: #4CAF50; color: white; padding: 5px; border: 1px solid #A3BFFA; }
    QLineEdit, QTextEdit { background-color: #FFFFFF; border: 1px solid #A3BFFA; border-radius: 5px; padding: 5px; font: 14px "Arial"; placeholder-text-color: #888; }
    QLineEdit:focus, QTextEdit:focus { border: 2px solid #4C6EF5; }
    QPushButton { background-color: #4CAF50; color: white; border: none; border-radius: 5px; padding: 8px; font: 14px "Arial"; min-width: 80px; }
    QPushButton:hover { background-color: #45A049; }
    QPushButton#logoutButton { background-color: #f44336; }
    QPushButton#logoutButton:hover { background-color: #da190b; }
    QComboBox { background-color: #FFFFFF; border: 1px solid #A3BFFA; border-radius: 5px; padding: 5px; font: 14px "Arial"; min-width: 100px; }
    QSpinBox { background-color: #FFFFFF; border: 1px solid #A3BFFA; border-radius: 5px; padding: 5px; font: 14px "Arial"; }
    QLabel#profileStatusLabel { font: bold 14px "Arial"; color: #2D6A4F; background-color: #D1E8E2; padding: 5px; border-radius: 5px; }
   </string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>760</width>
     <height>760</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="styleSheet">
        <string notr="true">font: bold 24px "Arial"; color: #2D6A4F; background-color: #D1E8E2; border-radius: 5px; padding: 10px;</string>
       </property>
       <property name="text">
        <string>Job Seeker Dashboard</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="profileStatusLabel">
       <property name="text">
        <string>Profile Verification Status: Not Verified</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_1">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>15</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="profileLabel">
       <property name="styleSheet">
        <string notr="true">font: bold 18px "Arial"; color: #2E4053; background-color: #D1E8E2; padding: 5px;</string>
       </property>
       <property name="text">
        <string>Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="profileForm">
       <property name="labelAlignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="formAlignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
       <property name="horizontalSpacing">
        <number>10</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="fullNameLabel">
         <property name="text">
          <string>Full Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="fullNameLineEdit">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Enter Full Name (e.g., John Doe)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="locationLabel">
         <property name="text">
          <string>Location:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="locationLineEdit">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Enter Location (e.g., Nairobi, Kenya)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="phoneLabel">
         <property name="text">
          <string>Phone:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="phoneLineEdit">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Enter Phone Number (e.g., +254 712 345 678)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="educationLabel">
         <property name="text">
          <string>Education:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTextEdit" name="educationTextEdit">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>60</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Enter Education (e.g., B.Sc. Computer Science, University of Nairobi, 2020)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="skillsLabel">
         <property name="text">
          <string>Skills:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTextEdit" name="skillsTextEdit">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>60</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Enter Skills (e.g., Python, SQL, Communication)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="workExperienceLabel">
         <property name="text">
          <string>Work Experience:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QTextEdit" name="workExperienceTextEdit">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>60</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Enter Work Experience (e.g., Software Developer at XYZ Corp, 2021-2023)</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="updateProfileButton">
         <property name="text">
          <string>Update Profile</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>15</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="jobsLabel">
       <property name="styleSheet">
        <string notr="true">font: bold 18px "Arial"; color: #2E4053; background-color: #D1E8E2; padding: 5px;</string>
       </property>
       <property name="text">
        <string>Available Jobs</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="searchSortLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLineEdit" name="searchLineEdit">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Search by Title or Company</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sortComboBox">
         <item>
          <property name="text">
           <string>Sort by Salary (Asc)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sort by Salary (Desc)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sort by Deadline (Asc)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sort by Deadline (Desc)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="jobsTable">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="columnCount">
        <number>7</number>
       </property>
       <column>
        <property name="text">
         <string>Job ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Title</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Company</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Location</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Salary Range</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Deadline</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Action</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="applyLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLineEdit" name="jobIdLineEdit">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Enter Job ID to Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="applyButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="applicationsLabel">
       <property name="styleSheet">
        <string notr="true">font: bold 18px "Arial"; color: #2E4053; background-color: #D1E8E2; padding: 5px;</string>
       </property>
       <property name="text">
        <string>My Applications</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="applicationsTable">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <column>
        <property name="text">
         <string>Job ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Title</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Company</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>15</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="logoutButton">
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>logoutButton</sender>
   <signal>clicked()</signal>
   <receiver>DashboardSeeker</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>759</x>
     <y>689</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>699</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>